lumeer-remote/src/main/java/io/lumeer/remote/rest/OrganizationService.java

```diff
@@ -186,4 +190,11 @@ public CompanyContact setCompanyContact(@PathParam("organizationCode") final Str
       return companyContactFacade.setCompanyContact(organizationFacade.getOrganization(organizationCode), companyContact);
    }
 
+   @GET
+   @Path("convert/views")
+   public Response convertViews() {
+      viewConvertFacade.convertViews();
+      return Response.ok().build();
```

Comment:
When is this endpoint called?

Reply:
This is temporary endpoint for converting old query structure to new one.

Reply:
This will be deleted together with `ViewConvertFacade` after we convert Devel and Production database.